src/client/search/categories/categoryFiltersReducer.spec.js

```diff
@@ -0,0 +1,199 @@
+import { should } from 'chai';
+import deepFreeze from 'deep-freeze';
+import footballCategoryFiltersReducer from './categoryFiltersReducer';
+import * as actions from './categoryFilterActions';
+
+// copied from ./footballCategoryFiltersReducer
+const footballCategoryFiltersInitialState = {
```

Comment:
Rather than copying, could we export the default from footballCategoryFiltersReducer and import it here? That way we can never mismatch them